.github/workflows/nolints.yml

```
@@ -47,15 +47,15 @@ jobs:
 
       - name: build mathlib
         id: build
-        uses: liskin/gh-problem-matcher-wrap@v2
+        uses: liskin/gh-problem-matcher-wrap@v3
         with:
           linters: gcc
           run: |
-            bash -o pipefail -c "env LEAN_ABORT_ON_PANIC=1 lake build -KCI | tee stdout.log"
+            bash -o pipefail -c "env LEAN_ABORT_ON_PANIC=1 lake build --wfail -KCI"
 
-      - name: lint mathlib
+      - name: update nolints.json and style-exceptions.txt
```

```suggestion
      - name: annotate the PR, update nolints.json and style-exceptions.txt
```

---

Actually, no, your description is complete wrong: it does not update any file, it only recreates `nolints.json` and `style-exceptions.txt` for the purpose of diffing them with the existing ones.
```suggestion
      - name: annotate the PR with linting and style errors
```

---

After #13343, this will be correct: let's just land that PR first.

---

That PR has landed, so this hunk is now removed.